doc/Developing.md

```diff
@@ -235,33 +235,61 @@ etc.) by opening `packages/<package name>/coverage/index.html`.
 
 ## Building on Windows
 
-Run cmd.exe as an administrator and install `choco` by copy-pasting the command
-to your console:
+1. Run cmd.exe as an Administrator. You can do this by right-clicking "Command Prompt" in the Windows Start Menu, and then right-clicking "Run as Administrator".
 
-    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
-
-Install `yarn` via `choco`. The `yarn` installation ensures that you will have
-Node.js and npm too:
-
-    choco install yarn
-
-Install `git` via `choco`
-
-    choco install git
-
-Install the correct version of `yarn` (The version is important)
+2. Install `choco` by copy and pasting the command below into the Command Prompt, and then pressing the `return` key to run it:
 
-    choco install yarn --version 1.7.0 -y
-
-Install Windows-Build-Tools.
-Run PowerShell as an administrator and copy-paste the below command:
-
-    npm --add-python-to-path install --global --production windows-build-tools
-
-Clone, build and run Theia.
-Using Git Bash as administrator:
-
-    git clone https://github.com/theia-ide/theia.git && cd theia && yarn && cd examples/browser && yarn run start
+```ps
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+3. Refresh the environment settings by running the following command in the Command Prompt: 
+```ps
+refreshenv
+```
+
+4. Install NodeJS via. choco (**NOTE**: the version is important!) by running the following in the Command Prompt:
+```ps
+choco install nodejs.install --version 8.11.4 -y
+```
+
+5. Install Yarn via. choco (**NOTE**: the version is important!) by running the following in the Command Prompt:
+```ps
+choco install yarn --version 1.7.0 -y
+```
+
+6. Install Git via. Choco:
```

Comment:
Choco -> choco (or `choco`).

Please if you escape something or put it between the double quotes, use it consistently.
"Command Prompt", Command Prompt
Choco, choco, `choco`.